opencti-platform/opencti-graphql/src/utils/stix-filtering/stix-filtering.ts

```diff
@@ -0,0 +1,71 @@
+import { INDICATOR_FILTER, } from '../filtering';
+import { getStixTesterFromFilterKey } from './stix-testers';
+import { testFilterGroup } from './boolean-logic-engine';
+import type { Filter, FilterGroup } from './filter-group';
+import { getEntitiesMapFromCache } from '../../database/cache';
+import { isUserCanAccessStixElement, SYSTEM_USER } from '../access';
+import { ENTITY_TYPE_RESOLVED_FILTERS } from '../../schema/stixDomainObject';
+import type { AuthContext, AuthUser } from '../../types/user';
+import type { StixId, StixObject } from '../../types/stix-common';
+import { extractStixRepresentative } from '../../database/stix-representative';
+
+// TODO: changed by Cathia for #2686, to integrate properly next
+const LABEL_FILTER = 'objectLabel';
```

Comment:
Note: other keys have been changed (ex: objectMarking instead of markedBy)

Reply:
I've checked and I updated all the keys I'm using for now (the ones that were handled by isStixMatchFilter).
Obviously, when we will extend to other keys I'll make sure to use the new keys.